src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/navigation/local/LocalNavigationMobile.java

```diff
@@ -0,0 +1,122 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.navigation.local;
+
+import com.wikia.webdriver.common.logging.Log;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class LocalNavigationMobile extends WikiBasePageObject {
+
+  @FindBy(css = ".wds-community-bar")
+  private WebElement communityBar;
+
+  @FindBy(css = ".wds-community-bar .wds-community-bar__sitename")
+  private WebElement communityName;
+
+  @FindBy(css = ".wds-community-bar__discussions")
+  private WebElement discussionsEntrypoint;
+
+  @FindBy(css = ".wds-community-bar__navigation")
+  private WebElement navigationEntrypoint;
+
+  @FindBy(css = ".wds-dropdown__content")
+  private WebElement navigationToogleDropdown;
+
+  @FindBy(css = ".wds-community-bar__level-1 ")
+  private WebElement subMenu;
+
+  @FindBy(css = ".wds-community-bar__level-1 > li > a")
+  private List<WebElement> firstLevelMenuLinks
+      ;
+
+  @FindBy(css = ".wds-dropdown__content .wds-community-bar__navigation-back")
+  private WebElement backButton;
+
+  @FindBy(css = ".wds-community-bar__navigation-header")
+  private WebElement secondLevelMenuHeader;
+
+
+  private boolean isElementVisible(WebElement element) {
+    try {
+      return element.isDisplayed();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)